NitroxModel/DataStructures/GameLogic/Entities/Bases/BuildEntity.cs

```diff
@@ -30,7 +30,7 @@ public static BuildEntity MakeEmpty()
 
     /// <remarks>Used for deserialization</remarks>
```

Comment:
I don't think this comment would be too relevant because we could come to a point when we want to use those contructors. For example, we already do it with PlayerWorldEntity